rosidl_generator_c/cmake/rosidl_generator_c_generate_interfaces.cmake

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 find_package(rcutils REQUIRED)
+find_package(rosidl_cmake REQUIRED)
```

Comment:
Since we are adding this here, we should also add a dependency in `package.xml`.  I think that one needs to be a `<build_export_depend>`.

Reply:
Fixed in a0dca0c6781a9c23618ee1383febccc37d957f95.